content/coronavirus_business_page.yml

```diff
@@ -29,6 +29,12 @@ content:
       href: /government/news/coronavirus-support-for-employees-benefit-claimants-and-businesses
     - text: If you're self employed, you'll get up to £2,500 a month in grants, for up to 3 months
       href: /government/news/chancellor-gives-support-to-millions-of-self-employed-individuals
+  related_links:
+    list:
```

Comment:
Should we add related link title?

Reply:
Can do? It was hardcoded in the partial so I left it out, but will update.